src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -622,10 +623,14 @@ public function collectFromCode($content, $fileName, Module $module)
             $previousToken = $thisToken;
             $thisToken = $token;
             if (is_array($token)) {
-                list($id, $text) = $token;
+                list($id, $text, $lineNo) = $token;
+                // minus 2 is intentional
+                $line = $lines[$lineNo - 2] ?? '';
 
                 // Collect use statements so we can get fully qualified class names
-                if ($id === T_USE) {
+                // Note that T_USE will match both use statements and anonymous functions with the "use" keyword
+                // e.g. $func = function () use ($var) { ... };
+                if ($id === T_USE && !preg_match('#function *\(#', $line)) {
```

Comment:
This falls apart on:
```php
function ($lots, $of, $args)
    use ($someVar) {
        // Something here
}
```
or
```php
function (
    $lots,
    $of,
    $args
) use ($someVar) {
        // Something here
}
```
That's an edge case for sure, but when lines get too long it could definitely happen.

Probably just check for `use (` instead? i.e. if there's a bracket instead of an alphanumeric character following the `use`, then it's not a `use MyClass;` statement.

Please also include one of the above examples in a test as well.

Reply:
Updated